config/routes.rb

```diff
@@ -1,9 +1,8 @@
 Rails.application.routes.draw do
   resources :status, only: [:index]
-  resources :proceeding_types, only: [:index]
 
   namespace 'v1' do
-    resources :status, only: [:index]
```

Comment:
Is it intentional that you are removing `/v1/status` as well as adding `/v1/proceeding_types`?

Reply:
I had a chat with Sergio, load balance will call /status not /v1/status, so I removed it.
proceeding types are part of API which is at the moment represented as /v1 , so makes sense to change it like /v1/proceeding_types

Reply:
I think I would have done two commits as you suggested but I did one commit, tried to explain 2 changes in one commit message: "added /v1 in request path for proceeding types and removed status from v1 namespace because (it exist outside v1 namespace which is the correct location)"